pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -49,10 +52,30 @@ var orderedCharts = []string{
 	"istio/charts/kiali",
 }
 
+const (
+	smcpTemplatePath        = "/etc/istio-operator/templates/"
+	smcpDefaultTemplatePath = "/etc/istio-operator/default-templates/"
+	smcpDefaultTemplate     = "default"
+)
+
 func (r *ControlPlaneReconciler) Reconcile() (reconcile.Result, error) {
 	var err error
 	var ready bool
 
+	//render the ServiceMeshControlPlane templates
+	spec, err := r.updateSMCPWithTemplates(r.Instance.Spec)
```

Comment:
I think this should be moved into `renderCharts()`.

Reply:
We should probably only be doing this when we render the charts, which is not on every reconcile, as we may be coming back after pausing for a component to become available.

Reply:
Done